src/server/server.js

```diff
@@ -91,8 +87,10 @@ const fastify = createServer({
   logger: true,
 })
 
+fastify.get('/healthcheck', async () => {})
```

Comment:
Two small nits:

- could we add a basic test for this endpoint?
- does this handler need to be async?

Reply:
Yes, I guess, but.... adding the test infrastructure for this CLI is a lot of work, and when we actually test the fly deployment, this is implicitly tested, because Fly does not consider the app deployed until this returns 200.

In terms of handler being async, if it's not then it needs to be `(req, reply) => reply.send()`. Writing async is shorthand to avoid that (it replies with the return value of the function)